README.md

```diff
@@ -46,7 +46,7 @@ async def main() -> None:
         listener_id = await client.register_event_listener()
 
         while True:
-            events = await client.fetch_event_listener(listener_id)
+            events = await client.fetch_events()
```

Comment:
We fetch "events" not a "listener".

Reply:
True, but this is a breaking change. I don't think that anyone is using this package yet, but would be good to mention this somewhere in the changelog.